src/systemd/sd-event.h

```diff
@@ -91,15 +91,15 @@ sd_event* sd_event_unref(sd_event *e);
 int sd_event_add_io(sd_event *e, sd_event_source **s, int fd, uint32_t events, sd_event_io_handler_t callback, void *userdata);
```

Comment:
`s -> ret`
and below.

Reply:
Sure, I can mass rename this header file and functions to keep it consistent. I didn't want to needlessly mass rename things

Reply:
Still I'd like to rename `s` -> `ret` consistently in the commit.